### PR TITLE
Feature: Adds SVM Content

### LINKED ISSUE #7: Create required content for "SVM"
### To Do
- [x] Create nescessary content/page for "More Algorithms: SVM" [ML Deep-Dive]
- [x] Create Github page for SVM Task

#### Instructions for students performing task
- Find a suitable dataset for the task
- Implement SVM
- Post results and a short write-up in Slack

#### Resources
- [In-Depth: Support Vector Machines](https://jakevdp.github.io/PythonDataScienceHandbook/05.07-support-vector-machines.html)

### PR DESCRIPTION
**Pull request by valiantone**
## Background
Content for demonstration and task pages is required in the section **More Algorithms** as part of the **ML Deep Dive**.

Especially felt like there should be an easy + short summarative explanation of the "kernel trick" since this is the selling point of SVMs.
 
### Relevant Tickets
* [DSA-7](https://github.com/TechLabs-Berlin/data-science-accelerator/issues/7)

### Changes
- Creates content page for "More Algorithms: SVM"
- Adds instructional page for SVM Task

### CODE REVIEW

**Comment by valiantone** [comment]
@Maxscha @MerlinSchaefer ready for review

File: ml-deep-dive/svm-task.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# SVM: Task
+
+1. Find an appropriate dataset for this task
+2. Implement SVM on this dataset and push the results to your public Github repo.
+3. Post results in this [Slack thread](https://techlabs-community.slack.com/archives/G01LXNY9F6J/p1612527408007600) with a few words describing your solution.
```
Comment (Maxscha) [inline ml-deep-dive/svm-task.md:5]: Wrong slack thread, use this one: https://techlabs-community.slack.com/archives/G01LXNY9F6J/p1613226232000500

File: ml-deep-dive/svm.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
+# Support Vector Machines
+Support vector machines (SVMs) are another subset of supervised learning methods that can be used for either classification or regression tasks. To find out what makes Support Vector Machines unique, and get a more expansive overview of SVMs, we would encourage students to follow the tutorial linked below:
```
Comment (Maxscha) [inline ml-deep-dive/svm.md:2]: I am not sure, is the link optional now or not? I would say yes, but if not I would put it into Optional in the bottom.

File: ml-deep-dive/svm.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+1. SVMs are highly effective in high-dimensional spaces, making them one of the most frequently used solutions for dimensional data.
+2. SVMs tend to generally find an optimal solution due to underlying convex optimization.
+3. SVMs can deal with linearity or non-linearity in the data, by choice of appropriate kernels
+4. SVMs make use of the the **Kernel Trick** to handle non-linearly separable data
+### Kernel Trick
+> A kernel is simply a transformation of a two-dimensional plane to a higher dimensional space.
+
+Sometimes separability cannot be achieved in a linear space. By application of a kernel trick, non-linear data is projected in to a higher dimensional space, where separation is much easier.
+
+![kernel-trick](./svm_kernel_trick.jpg)
```
Comment (Maxscha) [inline ml-deep-dive/svm.md]: I would suggest making a new folder and putting it in their :)

File: ml-deep-dive/svm-task.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+4. Comment, discuss and engage with other solutions in the Slack thread.
+
+### Example Dataset
+A dataset has been attached for reference:
+- [Data](./bill_authentication.csv)
```
Comment (Maxscha) [inline ml-deep-dive/svm-task.md]: Would create a asset folder and put our assets in their :)

**Review by Maxscha — COMMENTED** [review]
Great content, just some small things which should be changes. Thank you :)

File: ml-deep-dive/svm-task.md (hunk 1 above)
Comment (valiantone) [reply ml-deep-dive/svm-task.md:5]: - [x] Fix Slack thread link

File: ml-deep-dive/svm.md (hunk 2 above)
Comment (valiantone) [reply ml-deep-dive/svm.md:2]: Yes, still recommended that students go through this tutorial, it is **not optional**. All optional reading is in the end.

File: ml-deep-dive/svm.md (hunk 3 above)
Comment (valiantone) [reply ml-deep-dive/svm.md]: - [x] Move images to dedicated folder

File: ml-deep-dive/svm-task.md (hunk 4 above)
Comment (valiantone) [reply ml-deep-dive/svm-task.md]: - [x] Move CSV and data assets to "assets"

File: ml-deep-dive/svm.md (hunk 3 above)
Comment (MerlinSchaefer) [reply ml-deep-dive/svm.md]: lgtm (once image is in asset folder)

File: ml-deep-dive/svm.md (hunk 2 above)
Comment (MerlinSchaefer) [reply ml-deep-dive/svm.md:2]: Would not make it optional either, SVMs are too important imo

**Review by MerlinSchaefer — COMMENTED** [review]
Nice work! Thanks for jumping in!

**Pull request merged**